login.html

```diff
@@ -24,21 +25,23 @@ <h1 class="m-3">Slaps!</h1>
           <label class="m-3">Search a song or artist</label>
           <input class="form-control text-center" id="searched" placeholder="For Whom the Bell Tolls">
           <br>
-          <button type="button" id="searchBtn" class="btn btn-outline-danger m-2">Song</button>
-          <button type="button" id="searchBtn" class="btn btn-outline-danger m-2">Artist</button>
+          <button type="button" id="searchBtn" class="btn btn-outline-danger m-2 search">Song</button>
+          <button type="button" id="searchBtn2" class="btn btn-outline-danger m-2 search">Artist</button>
+
         </div>
         </form>
     </div>
-    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
-      integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
       integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
       crossorigin="anonymous"></script>
 
     <script>
-      $("#searchBtn").on("click", function(){
+      $("#searchBtn").on("click", function (event) {
+        event.preventDefault()
         let songSearched = $("#searched").val();
         localStorage.setItem("songName", songSearched)
+        //location.assign("/C:/Users/tfi10/Desktop/repos/Slaps/main.html?search=" + songSearched)
```

Comment:
I commented this line out for now because this URL is specific to your computer, and as such it will only work on your system. Try to see if you can find a way to redirect without using a hardcoded path